packages/fast-usdc/src/fast-usdc-policy.core.js

```diff
@@ -18,11 +19,13 @@ const FEED_POLICY = 'feedPolicy';
  * XXX copied from fast-usdc.start.js
  *
  * @param {ERef<StorageNode>} node
- * @param {FeedPolicy} policy
+ * @param {ERef<Marshaller>} marshaller
+ * @param {FeedPolicy & Passable} policy
  */
-const publishFeedPolicy = async (node, policy) => {
+const publishFeedPolicy = async (node, marshaller, policy) => {
```

Comment:
can `policy` have remotables in it? if not, you can use a data-only marshaller that's local to this function.